handlers.go

```diff
@@ -73,6 +76,40 @@ func compareVersions(a, b string) int {
 	return 0
 }
 
+func tBirdSha1Product(productSuffix string) string {
+	switch productSuffix {
+	case "beta", "beta-latest":
+		return tBirdWinXPLastBeta.Version
+	case "ssl":
+		return tBirdWinXPLastRelease.Version + "-ssl"
+	case "latest":
+		return tBirdWinXPLastRelease.Version
+	}
+
+	productSuffixParts := strings.SplitN(productSuffix, "-", 2)
+	ver := productSuffixParts[0]
+
+	possibleVersion := tBirdWinXPLastRelease
+	if strings.Contains(ver, ".0b") {
+		possibleVersion = tBirdWinXPLastBeta
+	}
+
+	if compareVersions(ver, possibleVersion.Version) == -1 {
+		return productSuffix
+	}
+
+	if len(productSuffixParts) == 1 {
+		return possibleVersion.Version
+	}
+
+	switch productSuffixParts[1] {
+	case "ssl":
+		return possibleVersion.Version + "-ssl"
+	}
```

Comment:
This single-case `switch` could be simplified to just an `if productSufficsParts[1] == "ssl" {`